backstop_data/engine_scripts/puppet/page-objects/crm-page.js

```diff
@@ -27,15 +27,16 @@ module.exports = class CrmPage {
    * @param {String} label
    */
   async clickSelect2Option(selector, label) {
-    await this.engine.click(`${selector} [class^="select2-choice"]`);
-    await this.engine.evaluate(label => {
+    await this.engine.evaluate((label, selector) => {
+      document.querySelector(`${selector} [class^="select2-choice"]`).click();
```

Comment:
it's not really clear from the PR description why this is needed. Can you provide more details please

Reply:
*Reason* -

Initially Scenario "Standard Search - with many options" throws *Can't dispatch event of null." So after debugging I found it once pupeteer after clicking on the first option and selecting the option, it can't click the dropdown open for the second time. It also made the code wait for the selector to be visible before clicking on it, but it din't happen.
It even clicks on the button but somehow it is not opening the dropdown.

When I try to pass the selector through plain Vanilla JS click() function, it was able to make the dropdown visible and hence able to select the 2nd and 3rd options.